firefoxThemes/modules/dom/inits.js

```diff
@@ -0,0 +1,49 @@
+//Place functions that needs to activate right away here!
+import {isSpecificSysTheme, isSysDark, isSysDefault} from "../modes/system/system.js";
+
+function populateSelect(selectEl, systemTheme, systemThemeChoice, waifuThemes) {
```

Comment:
I know this was moved from popup.js, but going forward, please start using more gender neutral variable names `waifuThemes`, `dokiThemes` works (you don't have to rename them right know, I'll probably do that when I work on that release.) 

Probably around Q2 2022, I'm going to add a few male themes (I know there is Astolfo, but he don't count), so `waifuThemes` will a bit of a misnomer 😄

Reply:
Interesting. With the name change from `waifuThemes` to `dokiThemes`, since `waifuThemes` is being used by previous users in their local storage, keep in mind that waifuThemes must be removed completely first before using the new name. *This is just a note.*

Reply:
Remove completely as in delete `waifuThemes` from local storage for users that are currently using Doki Theme. Then, replace it with the new name.